content/docs/tech-insights/data-sources/index.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Data Sources
+publishedDate: '2022-11-15'
+description: Managing Data Sources.
+---
+
+## Introduction
+
+Data Sources periodically collect Facts about your services from third-party vendors or arbitrary APIs. The Facts collected by Data Sources are used for defining Checks. Roadie comes with a handful of built-in Data Sources with predefined Facts that you can use without any setup. You can also define your own Data Sources, which are will use APIs or files in a repository as a source of truth.
+
+To manage Data Sources, go to Tech Insights → Data Sources.
+
+### Using Built-in Data Sources
+
+By default, Roadie includes 5 data sources for common vendors such as Datadog, Snyk, and GitHub. All built-in Data Sources have pre-defined Facts that you can use for defining your Checks.
+
+To use built-in Data Sources, you don’t need to do any set up. Jump to the next section to learn more about Checks. Note that you can’t delete nor trigger manual data updates on built-in Data Sources.
```

Comment:
'Checks' as clickable link?

Reply:
'Note that you can’t delete nor trigger manual data updates on built-in Data Sources.' 
Feels out of place here. Maybe move to the section where the triggering is explained?